Function/BookFunc/Models/RequestModel.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Text;
+
+namespace TeamsMeetingBookFunc.Models
+{
+    [SuppressMessage("Design", "CA1051:Do not declare visible instance fields", Justification = "Data Transfer Object")]
+    public class RequestModel
+    {
+        public DateTime? StartDateTime;
+        public DateTime? EndDateTime;
+        public string Subject;
+        public string PatientEmailAddress;
+        public string DoctorEmailAddress;
+        public List<string> OptionalAttendees;
+    }
```

Comment:
We need a "CreateEvent" boolean parameter here, to pilot the Event creation. If not specified, the Event shouldn't be created

Reply:
I Agree